modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserServiceTest.java

```diff
@@ -214,6 +231,69 @@ public void testUpdateOrganizationsWithoutPermission() throws Exception {
 		}
 	}
 
+	@Test
+	public void testWorkflowTaskIndexedAfterCreatingUser() throws Exception {
+		String originalName = PrincipalThreadLocal.getName();
+
+		PermissionChecker originalPermissionChecker =
+			PermissionThreadLocal.getPermissionChecker();
+
+		try {
+			PrincipalThreadLocal.setName(TestPropsValues.getUserId());
+
+			PermissionChecker adminPermissionChecker =
+				PermissionCheckerFactoryUtil.create(TestPropsValues.getUser());
+
+			PermissionThreadLocal.setPermissionChecker(adminPermissionChecker);
+
```

Comment:
@victorcdp I don't get why we are setting the permissionChecker to the admin. Is not the same of the user that is running the test?
If it's really needed can we use the `ContextUserReplace contextUserReplace` method?

Reply:
It's because this class doesn't have a setup and no permission checker is initiated, the other tests in this class do the same thing